plugins/polio/js/src/domains/Chronogram/Chronogram/Table/useChronogramTableColumns.tsx

```diff
@@ -28,6 +28,7 @@ export const useChronogramTableColumns = (
                 Header: formatMessage(MESSAGES.labelCampaignObrName),
                 id: 'round__campaign__obr_name',
                 accessor: 'campaign_obr_name',
+                width: 700,
```

Comment:
Shouldn't we use an adaptive size here rather than a fixed size? Using a fixed size might behave incorrectly on smaller screens

Reply:
I'll give the same answer as https://github.com/BLSQ/iaso/pull/1536/files#r1706565661